pages/guide.md

```diff
@@ -106,13 +107,17 @@ Federally operated domains do not all end in `.gov`, `.mil`, or `.fed.us`. Some
 
 ### What about domains that are only used to redirect visitors to other websites?
 
-These domains must enable port 443, use and enforce HTTPS, and follow all the same requirements and guidelines as domains used to host websites and APIs, including HSTS and preloading.
+These domains must enable port 443 and use properly configured HTTPS.
```

Comment:
What "enforce HTTPS" means for redirect domains is ambiguous here, and is what the subsequent answer now addresses, so I changed it to "properly configured".